demonstrations/tutorial_qsvt_hardware.py

```diff
@@ -0,0 +1,161 @@
+r"""How to implement QSVT on hardware
+=======================================
+
+The :doc:`Quantum Singular Value Transform (QSVT) </demos/tutorial_intro_qsvt>`
+is a quantum algorithm that allows us to perform polynomial
+transformations on matrices or operators, and is rapidly becoming
+a go-to algorithm for :doc:`quantum application research </demos/tutorial_apply_qsvt>`
+in the `ISQ era <https://pennylane.ai/blog/2023/06/from-nisq-to-isq/>`__.
+
+In this how-to guide, we will show how we can implement the QSVT
+subroutine in a hardware-compatible way, taking your application research
+to the next level.
+
+Angles calculation
+------------------
```

Comment:
```suggestion
Calculating angles
------------------
```
It simplifies it a bit.